src/shared/lib/oql/AccessorsForOqlFilter.ts

```diff
@@ -149,12 +149,25 @@ export default class AccessorsForOqlFilter
         );
     }
 
+    // public gene(d: Datum) {
+    //     if (
+    //         this.molecularAlterationType(d.molecularProfileId) ===
+    //         AlterationTypeConstants.STRUCTURAL_VARIANT
+    //     ) {
+    //         return (d as StructuralVariant).site1HugoSymbol;
+    //     }
+    //     return (d as any).gene.hugoGeneSymbol;
+    // }
+
     public gene(d: Datum) {
         if (
             this.molecularAlterationType(d.molecularProfileId) ===
             AlterationTypeConstants.STRUCTURAL_VARIANT
         ) {
-            return (d as StructuralVariant).site1HugoSymbol;
+            return [
+                (d as StructuralVariant).site1HugoSymbol,
+                (d as StructuralVariant).site2HugoSymbol,
+            ];
         }
         return (d as any).gene.hugoGeneSymbol;
```

Comment:
should probably return this as an array now and change the interface

Reply:
I don't want to do that because using array is not as performant and most of the time it's just a string